packages/cli/src/environments/variables/variables.service.ee.ts

```diff
@@ -3,7 +3,7 @@ import type { Variables } from '@db/entities/Variables';
 import { InternalHooks } from '@/InternalHooks';
 import { generateNanoId } from '@db/utils/generators';
 import { canCreateNewVariable } from './environmentHelpers';
-import { CacheService } from '@/services/cache.service';
+import { CacheService } from '@/services/cache/cache.service';
```

Comment:
The variables cache currently uses the regular `get` and `set` but could benefit from hashes too now

Reply:
From what I can tell, the access pattern for variables is always wholesale retrieval as a list, as opposed to querying specifically by variable ID, so this would require a deeper refactoring?

Reply:
You're correct, variables are always read in batches, so it makes sense to store them as a single object. I had forgotten this.